src/main/java/build/buf/protovalidate/internal/evaluator/EvaluatorBuilder.java

```diff
@@ -193,6 +191,36 @@ private void buildValue(
     processRepeatedConstraints(fieldDescriptor, fieldConstraints, forItems, valueEvaluator);
   }
 
+  private void processIgnoreEmpty(
+      FieldDescriptor fieldDescriptor,
+      FieldConstraints fieldConstraints,
+      boolean forItems,
+      ValueEvaluator valueEvaluatorEval)
+      throws CompilationException {
+    if (forItems
+        && fieldConstraints.getIgnoreEmpty()
+        && fieldDescriptor.getType() != FieldDescriptor.Type.MESSAGE) {
+      FieldDescriptor desc = fieldDescriptor;
+      if (desc.isRepeated()) {
+        DescriptorProtos.FileDescriptorProto.Builder bldr =
+            DescriptorProtos.FileDescriptorProto.newBuilder().setName("tmp");
+        bldr.addMessageTypeBuilder()
+            .setName(desc.getContainingType().getName())
+            .addField(desc.toProto().toBuilder().clearLabel());
+        Descriptors.FileDescriptor[] deps = {};
+        try {
+          desc =
+              Descriptors.FileDescriptor.buildFrom(bldr.build(), deps)
+                  .findMessageTypeByName(desc.getContainingType().getName())
+                  .findFieldByNumber(desc.getNumber());
+        } catch (Descriptors.DescriptorValidationException e) {
+          throw new CompilationException(e.toString());
+        }
```

Comment:
@pkwarren this is the ☠️ I was talking about. I could not find any other way to get the zero value of a repeated field's elements.